crates/uv-bench/benches/uv.rs

```diff
@@ -141,6 +141,9 @@ mod resolver {
         universal: bool,
     ) -> Result<ResolverOutput> {
         let build_isolation = BuildIsolation::default();
+        let extra_build_requires = uv_distribution::ExtraBuildRequires {
```

Comment:
I think you should implement `Default` on `uv_distribution::ExtraBuildRequires`, so this can just be `ExtraBuildRequires::default`.